README.md

```diff
@@ -1,169 +1,171 @@
-# Download-Artifact v3
+# `@actions/download-artifact`
 
-This downloads artifacts from your build
+Download [Actions Artifacts](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) from your Workflow Runs. Internally powered by [@actions/artifact](https://github.com/actions/toolkit/tree/main/packages/artifact) package.
 
 See also [upload-artifact](https://github.com/actions/upload-artifact).
 
-# What's new
+- [`@actions/download-artifact`](#actionsdownload-artifact)
+  - [v4 - What's new](#v4---whats-new)
+    - [Improvements](#improvements)
+    - [Breaking Changes](#breaking-changes)
+  - [Usage](#usage)
+    - [Inputs](#inputs)
+    - [Outputs](#outputs)
+  - [Examples](#examples)
+    - [Download Single Artifact](#download-single-artifact)
+    - [Download All Artifacts](#download-all-artifacts)
+    - [Download Artifacts from other Workflow Runs or Repositories](#download-artifacts-from-other-workflow-runs-or-repositories)
+  - [Limitations](#limitations)
+    - [Permission Loss](#permission-loss)
 
-- Download all artifacts at once
-- Output parameter for the download path
-- Port entire action to typescript from a runner plugin so it is easier to collaborate and accept contributions
 
-Refer [here](https://github.com/actions/download-artifact/tree/v2) for the previous version
+## v4 - What's new
 
-# Usage
+> [!IMPORTANT]
+> download-artifact@v4+ is not currently supported on GHES yet. If you are on GHES, you must use [v3](https://github.com/actions/download-artifact/releases/tag/v3).
```

Comment:
As soon as we publish v4 upload and download we should preemptively open up 2 issues to track support for GHES. Just so that people can have a link as it is inevitable _someone_ will open an issue.

Another idea would be to open GHES issues ahead of team, add a link to the `README` and that will be even easier for users to track

Reply:
This is a great idea, let's chat about it in our sync

Reply:
@konradpabjan Just stumbled about this after upgrading to v4 and facing the issue. Is there any issue where this is tracked? Could not find any....